Core/src/org/sleuthkit/autopsy/directorytree/FileSearchProvider.java

```diff
@@ -23,5 +23,7 @@
  */
 public interface FileSearchProvider {
 
+    public void showDialog(long dataSourceID);
```

Comment:
Do we need both showDialog?  Can we deprecate the old one and have it call the new one with a null value?

Reply:
Good idea, I will make that change. Thanks